### PR TITLE
chore: Create forgot password page

### PR DESCRIPTION
**Pull request by Wandarry**
Create forgot password page

Screenshot 2024-05-29 at 11 27 10

### CODE REVIEW

File: app/(tabs)/forgotPassword.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import { Mail } from "@/components/icons/mail";
+import { TextInput } from "@/components/uikit/Input/dumb/TextInput";
+import { BackIconButton } from "@/components/uikit/Buttons/BackIconButton";
+import { SolidLong } from "@/components/uikit/Buttons/SolidLong";
+
+export default function ForgotPassword() {
+  return (
+    <Box flex={1} justifyContent="center" bgColor="$white" gap={48} px={17}>
+      <Box position="absolute" top={50} left={17}>
+        <BackIconButton isDisabled={false} />
```
Comment (BOCOVO) [inline app/(tabs)/forgotPassword.tsx:11]: What is the default value ?

File: app/(tabs)/forgotPassword.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      </Box>
+      <Box gap={14} px={8}>
+        <Text w={"75%"} color="$black" fontWeight="$semibold" fontSize={25}>
```
Comment (BOCOVO) [inline app/(tabs)/forgotPassword.tsx]:
```suggestion
        <Text w="75%" color="$black" fontWeight="$semibold" fontSize={25}>
```
also works

File: app/(tabs)/forgotPassword.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        <Text color="$black" fontSize="$lg">
+          Entrez votre adresse mail, nous vous y enverrons un code.
+        </Text>
+      </Box>
+      <TextInput placeholder="Email" icon={Mail} />
+      <SolidLong
+        message="M'envoyer le code"
+        isLoading={false}
+        isDisabled={false}
+        onPress={() => console.log("Login")}
```
Comment (BOCOVO) [inline app/(tabs)/forgotPassword.tsx]:
```suggestion
        onPress={() => console.log("Send reset password request")}
```

File: app/(tabs)/forgotPassword.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        </Text>
+        <Text color="$black" fontSize="$lg">
+          Entrez votre adresse mail, nous vous y enverrons un code.
+        </Text>
+      </Box>
+      <TextInput placeholder="Email" icon={Mail} />
+      <SolidLong
+        message="M'envoyer le code"
+        isLoading={false}
+        isDisabled={false}
```
Comment (BOCOVO) [inline app/(tabs)/forgotPassword.tsx:25]: why you always add this prop 🤔  ?

**Review by BOCOVO — CHANGES_REQUESTED** [review]

File: app/(tabs)/forgotPassword.tsx (hunk 1 above)
Comment (Wandarry) [reply app/(tabs)/forgotPassword.tsx:11]: Default value ?

File: app/(tabs)/forgotPassword.tsx (hunk 4 above)
Comment (Wandarry) [reply app/(tabs)/forgotPassword.tsx:25]: isDisabled ?

**Comment by Wandarry** [comment]
J'ai quelques questions ici @BOCOVO . A part ça, c'est bon

File: app/(tabs)/forgotPassword.tsx (hunk 4 above)
Comment (BOCOVO) [reply app/(tabs)/forgotPassword.tsx:25]: Yes

File: app/(tabs)/forgotPassword.tsx (hunk 1 above)
Comment (BOCOVO) [reply app/(tabs)/forgotPassword.tsx:11]: The button is disabled as default?

**Pull request merged**